### PR TITLE
remove jquery

### PR DESCRIPTION
**Pull request by shilomangam**
## Before you request a review, please make sure to check the following boxes:

- [x] Explain what has changed.
- [x] Explain why the change is needed.
- [x] Added tests **NA**
- [x] Update documentation **NA**
- [x] Merge master with no conflicts.
- [x] Make sure the build is passing on ci.
- [x] If this pr closes any issue, please add a github link to close it.
- [x] If you changed something in the product design/UX, you got the approval of [Ben Benhorin](https://wix.slack.com/messages/@benb).

### Thanks for contributing :)

Removed jQuery from wix-style-react, replaced with plain JS methods.

[ISSUE 1389](https://github.com/wix/wix-style-react/issues/1389)

### CODE REVIEW

File: src/ToggleSwitch/ToggleSwitch.driver.js [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
     exists: () => !!element,
     click: () => ReactTestUtils.Simulate.change(toggleSwitch),
-    isChecked: () => $(toggleSwitch).is(':checked'),
+    isChecked: () => toggleSwitch.checked,
     isDisabled: () => isClassExists(element, 'disabled'),
     isXSmall: () => isClassExists(element, 'toggleSwitchXSmall'),
     isSmall: () => isClassExists(element, 'toggleSwitchSmall'),
     isLarge: () => !isClassExists(element, 'toggleSwitchXSmall') && !isClassExists(element, 'toggleSwitchSmall'),
-    fillColor: () => $(element).find('path').css('fill'),
+    fillColor: () => getComputedStyle(element.querySelector('path'), null).getPropertyValue('fill'),
```
Comment (shlomitc) [inline src/ToggleSwitch/ToggleSwitch.driver.js]: can remove the null

**Comment by shilomangam** [comment]
@shlomitc fixed, please review and approve/disapprove

**Comment by shilomangam** [comment]
@shlomitc could you merge it please?
Thanks!

**Comment by argshook** [comment]
@shilomangam it looks okay to me but i'm a little afraid it might break peoples builds. maybe you tried to run tests on some projects with this PR?

**Comment by shlomitc** [comment]
I suggest releasing it as a separate release and then it's really easy to see the effect of it and see failing projects.

Overall, looks like it shouldn't break anyone

**Comment by argshook** [comment]
@shlomitc you mean release with only this change?

**Comment by shlomitc** [comment]
this.change === this.pr

**Pull request merged**